hyperapp.d.ts

```diff
@@ -0,0 +1,149 @@
+declare module "hyperapp" {
+  export interface VNodeData {
+    [key: string]: any
```

Comment:
Is it VNodeData where [lifecycle methods](https://github.com/hyperapp/hyperapp/blob/master/docs/lifecycle-events.md) and [keys](https://github.com/hyperapp/hyperapp/blob/master/docs/keys.md) would go? Would be nice to demonstrate those here too.